build-system/common/esbuild-babel.js

```diff
@@ -77,42 +141,45 @@ function getEsbuildBabelPlugin(
     return read;
   }
 
-  function transformContents(filepath, contents, hash) {
+  async function transformContents(contents, hash, babelOptions) {
```

Comment:
`async` here just introduces another tick. If there's no `await`, there's no need for `async`.
```suggestion
  function transformContents(contents, hash, babelOptions) {
```

Reply:
For convenience I use `await` within the function

Reply:
Removed

Reply:
I think you confused `BabelCache.set` and `transformContents` (which this comment is on). There's no `await` inside `transformContents`.

Reply:
you are right!